common/upgets/api_resource_spot_finder.lua

```diff
@@ -11,6 +11,7 @@ function upget:GetInfo()
 		license = "GNU GPL, v2 or later",
 		layer = -999999,
 		enabled = true,
+		handler = true,
```

Comment:
AFAIK without `handler = true` you can just do `gadgetHandler:RemoveCallIn("Update")`.